specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json

```
@@ -105,7 +105,7 @@
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
         }
-    }
+      }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies/{policyName}": {
```

to keep consistency between resources
```suggestion
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/applicationGatewayWebApplicationFirewallPolicies/{policyName}": {
```